compiler-utils/src/main/java/com/squareup/anvil/compiler/internal/ClassReference.kt

```diff
@@ -144,3 +148,38 @@ public fun ClassReference.indexOfTypeParameter(parameterName: String): Int {
     }
   }
 }
+
+@ExperimentalAnvilApi
+public fun ClassReference.scopeOrNull(
+  module: ModuleDescriptor,
+  annotationFqName: FqName
+): FqName? {
+  return when (this) {
+    is Descriptor -> {
+      clazz.annotationOrNull(annotationFqName)
+        ?.scope(module)
+        ?.fqNameSafe
+    }
+    is Psi -> {
+      clazz.scopeOrNull(annotationFqName, module)
+    }
+  }
+}
+
+@ExperimentalAnvilApi
+public fun ClassReference.qualifiers(
+  module: ModuleDescriptor
+): List<AnnotationSpec> {
+  return when (this) {
+    is Descriptor -> {
+      clazz.annotations
+        .filter { it.isQualifier() }
+        .map { it.toAnnotationSpec(module) }
+    }
```

Comment:
The curly braces here and at many other places are redundant. We usually prefer not to have them, if they aren't required. It's not dangerous to omit them, because you use `when` as an expression. I'd be in favor of removing them everywhere where not needed. But it's not a hill I would die on.